bench-tps/src/bench.rs

```diff
@@ -35,7 +35,7 @@ use std::{
 
 // The point at which transactions become "too old", in seconds.
 const MAX_TX_QUEUE_AGE: u64 =
-    MAX_PROCESSING_AGE as u64 * DEFAULT_TICKS_PER_SECOND / DEFAULT_TICKS_PER_SLOT;
+    MAX_PROCESSING_AGE as u64 * DEFAULT_TICKS_PER_SLOT / DEFAULT_TICKS_PER_SECOND;
```

Comment:
:man_facepalming: that was easy :) 
why is MAX_PROCESSING_AGE not sufficient?

Reply:
I know 🤦‍♀ Sorry about that!
MAX_PROCESSING_AGE is in slots (despite the misleadlingly named MAX_HASH_AGE_IN_SECONDS in clock.rs), and bench-tps timing is in Duration, so we need to convert from slots to approximate seconds

Reply:
wow that is so misleading haha. It makes sense now. Thanks!

Reply:
DEFAULT_TICKS_PER_SLOT / DEFAULT_TICKS_PER_SECOND is seconds/slot?

B/c DEFAULT_TICKS_PER_SLOT < DEFAULT_TICKS_PER_SECOND, this will always come out to zero, no?

Reply:
Why 1 specifically?

Reply:
@carllin I'm going to merge this to get the fix in, but happy to add in a check in a separate pr.